nodewatcher/plugins/torque.py

```diff
@@ -50,6 +51,43 @@ def getJobs(hostname):
 
     return _jobs
 
+def queueHasPendingJobs():
+    command = "qstat -Q"
+    _command = shlex.split(command)
+    error = False
+    has_pending = False
+    try:
+        process = subprocess.Popen(_command, env=dict(os.environ),
+                                   stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError:
+        log.error("Failed to run %s\n" % command)
+        error = True
+
+    output = process.communicate()[0]
+    exit_code = process.poll()
+
+    if exit_code != 0:
+        log.error("Failed to run %s\n" % command)
+        error = True
+        return has_pending, error
+
+    lines = output.split("\n")
```

Comment:
A comment with an example of the output would be helpful:
```
Queue              Max    Tot   Ena   Str   Que   Run   Hld   Wat   Trn   Ext T   Cpt
----------------   ---   ----    --    --   ---   ---   ---   ---   ---   --- -   ---
batch                0      7   yes   yes     5     2     0     0     0     0 E     0
```

Reply:
Okay